detection-rules/risk_score.yml

```diff
@@ -0,0 +1,18 @@
+name: "Risk Score: Medium to High Confidence"
+description: |
+  Aggregates a series of MQL-based signals to determine risk of a message
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and beta.risk_score().confidence in ("medium", "high")
+  and (
+    (
+      profile.by_sender().prevalence in ("new", "outlier")
```

Comment:
can we drop this? we'll likely run into the FTS issues from the past

Reply:
@jkamdjou does this seem better to you?